packages/desktop-client/src/components/schedules/ScheduleDetails.jsx

```diff
@@ -321,7 +340,9 @@ export function ScheduleDetails({ modalProps, actions, id }) {
     };
   }, [state.schedule, state.transactionsMode, state.fields]);
 
-  const selectedInst = useSelected('transactions', state.transactions, []);
+  const selectedInst = useSelected('transactions', state.transactions, [
+    transaction ? transaction.id : null,
+  ]);
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I'm not entirely sure what problems a null value here might cause in other parts of the product. It would be safer to do this:

```suggestion
  const selectedInst = useSelected('transactions', state.transactions,  transaction ?  [transaction.id] : []);
```

Reply:
Like it makes, sense